engine/swarm/networking.md

```diff
@@ -158,6 +158,8 @@ the service a VIP. The VIP maps to a DNS alias based upon the service name.
 Containers on the network share DNS mappings for the service via gossip so any container on the network can access the service via its service
 name.
 
+>**Note**: Service discovery will only work if your services are attached to a user-created overlay network (see top of this article). When a Swarm is initialized an ingress network is created (if it doesn't already exist) however containers are not added directly to this network. Ingress is used to enable to routing mesh functionality of swarm mode.
```

Comment:
No need to  capitalize swarm

Maybe a little rewording:

When a swarm is initialized, an ingress network is created if it does not exist. This network is not used by containers, but to enable the daemons participating in the swarm to communicate.